lib/reporters/mini.js

```diff
@@ -200,7 +201,7 @@ MiniReporter.prototype.finish = function () {
 
 MiniReporter.prototype.write = function (str) {
 	cliCursor.hide();
-	this.currentStatus = str + '\n';
+	this.currentStatus = String(str);
```

Comment:
Well before it was concatenating with a newline. That's been removed so presumably you could just assign `this.currentStatus = str`?